frontend/src/audio.html

```diff
@@ -0,0 +1,84 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+templateEngineOverride: md, njk, html
+eleventyComputed:
+  title: "{{ audio.objectID }}"
+---
+<section class="flex flex-col m-10">
+  <div class="flex flex-col">
+    <div class="flex align-baseline justify-between">
+      <div>
+        <p class="font-bold text-lg">{{ audio.title }}</p>
+      </div>
+      <div>39:46</div>
+    </div>
+    <div class="flex flex-col md:flex-row justify-between mt-1">
+      <div class="flex self-start font-semibold text-xs">{{ audio.objectID }} | {{ audio.dateForHumans }} | {{ audio.location }} | Lecture | {{ audio.languages }}</div>
```

Comment:
Better use the same technique that is used in the search result item template. Namely, CSS borders to add separator between info blocks. The reason is that each block sould be an element with its own tootip (`title` attribute).

Reply:
code replaced with same technique as search result item templates, css borders and seperators.